cinema/views.py

```diff
@@ -12,6 +20,8 @@
     MovieDetailSerializer,
     MovieSessionDetailSerializer,
     MovieListSerializer,
+    OrderSerializer,
+    OrderListSerializer,
 )
```

Comment:
Code Style: It seems like you didn't add a blank line at the end of the import section. It's a good practice to separate import section from the rest of the code with a blank line.